### PR TITLE
Hedgedoc/update installation procedure

### PR DESCRIPTION
**Pull request by drupol**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by drupol** [comment]
@GrahamcOfBorg build hedgedoc

**Comment by drupol** [comment]
@GrahamcOfBorg build hedgedoc

File: pkgs/servers/web-apps/hedgedoc/default.nix [hunk 1]
```diff
@@ -73,15 +73,15 @@ mkYarnPackage rec {
 
     pushd node_modules/sqlite3
     export CPPFLAGS="-I${nodejs}/include/node"
-    npm run install --build-from-source --nodedir=${nodejs}/include/node
+    yarn install --frozen-lockfile
```
Comment (SuperSandro2000) [inline pkgs/servers/web-apps/hedgedoc/default.nix:76]: that is just for sqlite3 which shouldn't matter to much.

File: pkgs/servers/web-apps/hedgedoc/default.nix [hunk 2]
```diff
@@ -73,15 +73,15 @@ mkYarnPackage rec {
...
     popd
 
     pushd node_modules/esbuild
     rm bin/esbuild
     ln -s ${lib.getBin esbuild-hedgedoc}/bin/esbuild bin/
     popd
 
-    npm run build
+    yarn build
```
Comment (SuperSandro2000) [inline pkgs/servers/web-apps/hedgedoc/default.nix:84]: This basically runs the same build script and should be noop.

**Comment by SuperSandro2000** [comment]
Fixed it locally by removing the erroring file before bundling.

#182643

**Pull request closed without merging**